.github/workflows/ci.yml

```diff
@@ -11,26 +11,55 @@ on:
 
 env:
   NXF_ANSI_LOG: false
+  NFT_VER: "0.9.0"
   NXF_SINGULARITY_CACHEDIR: ${{ github.workspace }}/.singularity
   NXF_SINGULARITY_LIBRARYDIR: ${{ github.workspace }}/.singularity
+  # env var to force tests
```

Comment:
Can we get rid of these commented lines?

Reply:
yes. removed.